mocks/app.crunch.io/api/datasets/by_name/Vegetables%20example.json

```diff
@@ -23,8 +23,11 @@
             },
             "size": {
                 "columns": 23,
+                "derivations": 3,
                 "rows": 210,
-                "unfiltered_rows": 210
+                "source_columns": 20,
+                "unfiltered_rows": 210,
+                "variables": 23
```

Comment:
oops, sorry about this surprise recent change.
It’s really good to see it reflected in an updated fixture here though actually.

Reply:
Oh yeah, I see this as a major benefit of the scriptable regenerating .json files. If we needed to develop tests for a feature that depended on these, we would already have fixtures in place.